public/themes/custom/hdbt_subtheme/src/scss/06_components/paragraphs/_unit-ontologyword-details.scss

```diff
@@ -0,0 +1,26 @@
+
+.unit__ontologyword-details {
+  h5 {
+    margin-bottom: $spacing;
+  }
+}
+
+.unit__ontologyword-details__schoolyear {
+  @include heading-hyphens;
+  @include font('h4');
+  display: block;
+  padding-top: $spacing;
+  margin-top: $spacing;
+}
+
+.unit__ontologyword-details-row {
+  @include font('body');
+  margin-bottom: $spacing;
+
+  .unit__ontologyword-details-row__label {
+    @include font('h6');
+    align-items: flex-start;
+    display: flex;
+    margin-bottom: 8px;
```

Comment:
Should some variable be used here instead of pixels? For example `$spacing-half`?